### DISCUSSION TITLE
Add change password from mobile app and recover 2FA from app

### DISCUSSION THREAD

**Discussion by nandy2187-zz** [community/community#10500] [Mobile]
I'm signed in and I've lost access from my computer and it seems I have to request to delete my username to support because they are not available to help me even though I'm signed in the mobile app and I can verify I'm me

**Comment by brianlovin** [comment]
Hey @nandy2187-zz, really sorry to hear about the trouble here. Unfortunately we aren't going to be able to help with any kind of account recovery, the best thing to do here will be to continue working with the support team to verify your information and account access.
